test/test_datasource.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import os.path
+import functools
+
+from tableaudocumentapi import Datasource
+
+TEST_TDS_FILE = os.path.join(
+    os.path.dirname(__file__),
+    'assets',
+    'datasource_test.tds'
+)
+
+
+class DataSourceColumns(unittest.TestCase):
+    def setUp(self):
+        self.ds = Datasource.from_file(TEST_TDS_FILE)
+
+    def test_datasource_returns_correct_columns(self):
+        self.assertIsNotNone(self.ds.columns)
+        self.assertIsNotNone(self.ds.columns.get('[Number of Records]', None))
```

Comment:
the bracket names are the most precise names we have for columns and how we would refer to them in calculations, etc, so I think it's an okay overhead.